alerting/src/main/kotlin/org/opensearch/alerting/settings/AlertingSettings.kt

```diff
@@ -156,5 +158,18 @@ class AlertingSettings {
             -1L,
             Setting.Property.NodeScope, Setting.Property.Dynamic
         )
+
+        val ALERTING_TRIGGER_MAX_ACTIONS = Setting.intSetting(
+            "plugins.alerting.trigger.max_actions",
+            DEFAULT_ALERTING_TRIGGER_MAX_ACTIONS,
+            -1,
+            Setting.Property.NodeScope, Setting.Property.Dynamic
+        )
+        val ALERTING_TRIGGER_TOTAL_MAX_ACTIONS = Setting.intSetting(
+            "plugins.alerting.trigger.total_max_actions",
```

Comment:
Should we change the namespace and call it ``plugins.alerting.max_actions_across_triggers`` for clarity that this is total max across all the triggers?